easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb

```
@@ -0,0 +1,494 @@
+easyblock = 'PythonBundle'
+
+name = 'Python-bundle-PyPI'
+version = '2023.06'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages from PyPI"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('UnZip', '6.0'),
+    ('Rust', '1.70.0'),  # required for setuptools-rust, which is needed for cryptography and bcrypt
+    ('pkgconf', '1.9.5'),
+    ('git', '2.41.0', '-nodocs'),  # required for pbr
+    ('setuptools-rust', '1.6.0'),  # required for cryptography and bcrypt
+    ('hatchling', '1.18.0'),
+    ('poetry', '1.5.1'),
+    ('scikit-build', '0.17.6'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('binutils', '2.40'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.13'),
+    ('libreadline', '8.2'),
+    ('ncurses', '6.4'),
+    ('SQLite', '3.42.0'),
+    ('XZ', '5.4.2'),
+    ('GMP', '6.2.1'),  # required for pycrypto
+    ('libffi', '3.4.4'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('cryptography', '41.0.1'),
```

All these deps are provided by the base `Python` easyconfig and can be removed from the bundle.
```suggestion
    ('Python', '3.11.3'),
    ('GMP', '6.2.1'),  # required for pycrypto
    ('cryptography', '41.0.1'),
```